modules/RTC/TPCUtils.js

```diff
@@ -354,6 +354,11 @@ export class TPCUtils {
     replaceTrack(oldTrack, newTrack) {
         const mediaType = newTrack?.getType() ?? oldTrack?.getType();
         const track = newTrack?.getTrack() ?? null;
+        const isNewSecondaryLocalSource = FeatureFlags.isMultiStreamSupportEnabled()
```

Comment:
this variable name suggest that only two streams are supported, but that's not the case is it?

Reply:
changed the variable name to `isNewLocalSource`